index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Dambi Stuart Software Engineer Profile">
+  <title>Stuart Dambi Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/styles.css">
+</head>
+
+<body>
+  <!-- Header -->
+  <header class="header">
+    <nav class="header-nav">
+      <a href="#" class="nav-link"><img src="/assets/images/mobile/menu.svg" alt=""></a>
+    </nav>
+  </header>
+  <!-- End Header -->
+
+  <!-- Introduction Area -->
+  <section class="introduction">
+    <h1 class="title">
+      Dambi Stuart
+    </h1>
+    <p class="intro-description">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+      works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+    <button class="intro-button">
+      Start Collaboration
+    </button>
+  </section>
+  <!-- End Introduction Area -->
+
+  <section class="project-banner">
+    <!-- <img src="/assets/images/mobile/projectbanner.svg" width="320px" height="160px" alt=""> -->
+  </section>
+
+  <h1 class="projects-title">Projects</h1>
+
+  <section class="projects">
+    <div class="project-card">
+      <div class="project-image"></div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project2.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
```

Comment:
- [ ] The same here image icon in the button is not shown. The path is broken in the remote maybe it works in your VS code/ other editor live extension server locally. It may be tried by adding a dot `.` before starting the path. :+1: 

Example
```
<img src="./assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
```
***Your design***
![Screenshot from 2022-12-03 18-37-47](https://user-images.githubusercontent.com/65129228/205450215-5d23c12d-8fdb-4167-b5fa-d19a70c93042.png)